### PR TITLE
Reader: update small to compact in post-card Component

### PR DESCRIPTION
**Pull request by jancavan**
In https://github.com/Automattic/wp-calypso/pull/7142, we plan on using `.post-card` as a generic card component name for the new cards in Reader Refresh.

This component name is currently being by `small.jsx` so this needs to be renamed.

Test live: https://calypso.live/?branch=update/reader/small-to-post-card-compact

### CODE REVIEW

File: client/components/post-card/compact.jsx [hunk 1]
```diff
@@ -15,43 +15,43 @@ import safeImageUrl from 'lib/safe-image-url';
 import resizeImageUrl from 'lib/resize-image-url';
 import AuthorAndSite from './author-and-site';
 
-export function SmallPostCard( { translate, post, site, onPostClick = noop, onSiteClick = noop } ) {
-	const classes = classnames( 'post-card small', {
+export function CompactPostCard( { translate, post, site, onPostClick = noop, onSiteClick = noop } ) {
```
Comment (mtias) [inline client/components/post-card/compact.jsx]: Can we do this as a prop? It's the pattern we use for other components. `<PostCard compact />` which gets the `is-compact` class.

Comment (blowery) [inline client/components/post-card/compact.jsx]: the compact variant here has a different DOM than the non-compact one. Maybe `Small` was a better name?

**Comment by jancavan** [comment]
@mtias There also isn't currently a `<PostCard>` component. There is a class named `.post-card` which is now tied to `small.jsx`. The plan here is to be able to use the `.post-card` class for the [new card designs](https://github.com/Automattic/wp-calypso/pull/7142), thus the renaming.

**Comment by mtias** [comment]
@jancavan the post-card folder is what we consider to be the component. Not sure what the overall plan is, but if we the idea is that we'll have a smaller card with different markup that probably needs to be a separate component. On the other hand, if we don't have a non-compact post card at the moment, it just makes sense to rename the compact version as the default.

**Comment by jancavan** [comment]
We're going to just separate these out into different components: `reader-search-card`, `reader-stream-card`, `reader-related-card, `reader-author-and-site` so closing this one for now.

**Pull request closed without merging**